apps/windows/metro.config.js

```diff
@@ -10,6 +10,7 @@ const blockList = exclusionList([
 module.exports = makeMetroConfig({
   projectRoot: __dirname,
   resolver: {
+    resolverMainFields: ['main'],
```

Comment:
This basically just undos what the update does.

Reply:
Do we need to file a bug?

Reply:
I'm not entirely sure? We have the option to specify the config/override, so idk if it's really a bug, just that our config wasn't set up properly? I guess we can file something and see what they say.

Reply:
Opened https://github.com/microsoft/rnx-kit/issues/1060

Reply:
Updated code according to Tommy's suggestion.